openshift_tools/monitoring/pminfo.py

```diff
@@ -149,13 +155,23 @@ def get_sampled_data(metrics, interval, count=1):
 
     return results
 
-def get_metrics(metrics=None):
+def get_metrics(metrics=None, derived_metrics=None):
     '''Get a list of metrics and query pcp for their values
     '''
     try:
         pcp = PMInfo()
         pcp.connect()
+
+        if derived_metrics:
+            pcp.register_derived_metric(derived_metrics)
+
+            if not metrics:
```

Comment:
I think spacing is off here.  I think metrics should be spaced over to the left.